app/controllers/member/dashboard_controller.rb

```diff
@@ -2,6 +2,8 @@ class Member::DashboardController < Member::BaseController
 
   def index
     authorize!(:show, :member_dashboard)
+    authorize!(:index, List)
```

Comment:
Turns out if I do the above, it will only test the instance of list. However if list is empty then it's authorising against nothing. The solution I have is the one that required.